src/components/pages/doc.yml

```diff
@@ -11,5 +11,20 @@ commentsPanelTitle:
   en: You are viewing the document page in "comments" mode
   de: Du befindest dich im Kommentar-Modus
 commentsHeader:
-  en: Comment topics
-  de: Kommentar-Themen
+  en: Comments
+  de: Kommentare
+viewHistoryRestrictionTooltip_annonymousUser:
+  en: Viewing document history is only allowed for logged in users
+  de: Ansehen von vorherige Versionen ?? is nur für angemeldet Benutzern elaubt
+editRestrictionTooltip_annonymousUser:
+  en: Document editing is only allowed for logged in users
+  de: Bearbeitung von Dokumenten is nur für angemeldet Benutzern elaubt
+editRestrictionTooltip_openContribution:
+  en: Editing this document was disabled by a maintainer
+  de: Die Bearbeitung dieses Dokuments wurde von einem Maintainer deaktiviert
+editRestrictionTooltip_archive:
+  en: This document is archived
+  de: Dieses Dokument ist archiviert
```

Comment:
In order to understand this the user has to have the knowledge what the button would do when the document would not be archived. We could add more info:

This document cannot be edited because it is archived
Dieses Dokument kann nicht bearbeitet werden da es archiviert ist

Reply:
That's true. And it's a tooltip, we can afford to be clearer (with more text) in tooltips. Changed.